xapian-letor/feature/feature.cc

```diff
@@ -36,10 +39,17 @@ void
 Feature::operator=(const Feature & o)
 {
     LOGCALL_VOID(API, "Feature::operator=", o);
-    internal = o.internal;
+    feature_db = o.feature_db;
+    feature_query = o.feature_query;
+    feature_doc = o.feature_doc;
+    termfreq = o.termfreq;
+    inverse_doc_freq = o.inverse_doc_freq;
+    doc_length = o.doc_length;
+    collection_length = o.collection_length;
+    collection_termfreq = o.collection_termfreq;
 }
```

Comment:
You don't need to provide an implementation for a private constructor or method that should never be called. (In C++11 you apparently can also do `= delete;` in the class declaration, but I don't know if there are any cross-compiler problems with that.)